lib/common/convert_to_pdf.rb

```diff
@@ -9,7 +9,7 @@ def initialize(file)
     end
 
     def run
-      in_file = Common::FileHelpers.generate_clamav_temp_file(@file.read)
+      in_file = Common::FileHelpers.generate_random_file(@file.read)
```

Comment:
do we know if clamav was intentionally chosen for virus protection/scanning purposes of the file that we would no longer get/want?

Reply:
That is my one concern. I've asked around a little bit but haven't gotten a satisfying answer yet.

Reply:
I'll approve, and trust for you to cover your bases if you think it's worth investigating further :)